lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,21 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        case app_name
+        when '.'
+          ::File.basename(Dir.getwd)
+        when '/'
```

Comment:
@jodosha, @joneslee85 mentioned about this use case of having '/' as the path, however personally I don't think anyone would be generating the app under root nor is it a good practice (and most likely permission denied). What do you guys think? I can remove it if it doesn't make sense.